docs/feature_unicode.md

```diff
@@ -75,20 +75,24 @@ void qk_ucis_symbol_fallback (void) { // falls back to manual unicode entry
 }
 ```
 
-## Unicode Input methods
+## Input Modes
 
-Unicode input in QMK works by inputting a sequence of characters to the OS,
-sort of like macro. Unfortunately, each OS has different ideas on how Unicode is input.
+Unicode input in QMK works by inputting a sequence of characters to the OS, sort of like a macro. Unfortunately, the way this is done differs for each OS, so a corresponding input mode has to be set.
 
-This is the current list of Unicode input method in QMK:
+The following input modes are available:
 
-* __UC_OSX__: MacOS Unicode Hex Input support. Works only up to 0xFFFF. Disabled by default. To enable: go to System Preferences -> Keyboard -> Input Sources, and enable Unicode Hex.
-* __UC_OSX_RALT__: Same as UC_OSX, but sends the Right Alt key for unicode input
-* __UC_LNX__: Unicode input method under Linux. Works up to 0xFFFFF. Should work almost anywhere on ibus enabled distros. Without ibus, this works under GTK apps, but rarely anywhere else.
-* __UC_WIN__: (not recommended) Windows built-in Unicode input. To enable: create registry key under `HKEY_CURRENT_USER\Control Panel\Input Method\EnableHexNumpad` of type `REG_SZ` called `EnableHexNumpad`, set its value to 1, and reboot. This method is not recommended because of reliability and compatibility issue, use WinCompose method below instead.
-* __UC_WINC__: Windows Unicode input using WinCompose. Requires [WinCompose](https://github.com/samhocevar/wincompose). Works reliably under many (all?) variations of Windows.
+* **`UC_OSX`**: Mac OS X built-in Unicode hex input. Works up to 0xFFFF. To enable, go to _System Preferences > Keyboard > Input Sources_, add _Unicode Hex Input_ to the list (it's under _Other_), then activate it from the input dropdown in the Menu Bar. By default, this mode uses the left Option key (`KC_LALT`), but this can be changed by defining `UNICODE_OSX_KEY` with another keycode.
+* **`UC_LNX`**: Linux built-in IBus Unicode input. Works up to 0xFFFFF. Enabled by default and works almost anywhere on IBus-enabled distros. Without IBus, this mode works under GTK apps, but rarely anywhere else.
+* **`UC_WIN`**: _(not recommended)_ Windows built-in Unicode hex numpad input. To enable, create a registry key under `HKEY_CURRENT_USER\Control Panel\Input Method\EnableHexNumpad` of type `REG_SZ` called `EnableHexNumpad`, set its value to `1`, and reboot. This mode is not recommended because of reliability and compatibility issue; use the `UC_WINC` mode instead.
```

Comment:
I'd been meaning to add this for a while, now....

```suggestion
* **`UC_WIN`**: _(not recommended)_ Windows built-in Unicode hex numpad input. To enable this, create a registry key under `HKEY_CURRENT_USER\Control Panel\Input Method\EnableHexNumpad` of type `REG_SZ` called `EnableHexNumpad`, set its value to `1`, and reboot. This mode is not recommended because of reliability and compatibility issue; use the `UC_WINC` mode instead.
** This can be accomplished by running: `reg add "HKCU\Control Panel\Input Method" /v EnableHexNumpad /t REG_SZ /d 1` with administrative privilege. 
```

Reply:
Done!